service/src/main/kotlin/io/provenance/onboarding/frameworks/provenance/exceptions/Exceptions.kt

```diff
@@ -7,3 +7,4 @@ open class ResourceNotFoundException(message: String? = "", cause: Throwable? =
 open class WontRunLoanValidationException(message: String? = "", cause: Throwable? = null) : RuntimeException(message, cause)
 
 open class ExistingScopeNotChangedException(message: String) : RuntimeException(message)
+open class ContractTxException(message: String) : RuntimeException(message)
```

Comment:
Nit: Just curious, why are these exception classes `open`? Is the plan to extend them later?

Reply:
LOL copy and paste. I'll drop the open and we can put it back if anyone wants to...